app/renderer/components/common/commonForm.js

```diff
@@ -81,60 +86,65 @@ class CommonFormTextbox extends ImmutableComponent {
   }
 }
 
-class CommonFormClickable extends ImmutableComponent {
+class CommonFormSection extends ImmutableComponent {
   render () {
-    return <div className={css(styles.commonFormClickable)} {...this.props} />
+    return <div className={css(
+      styles.commonForm__section,
+      this.props.custom
+    )}
+      data-l10n-id={this.props.l10nId}
+      data-l10n-args={JSON.stringify(this.props.l10nArgs || {})}
```

Comment:
This does not work since the json string is escaped. Would anyone please help me to fix this? thanks!

<img width="482" alt="screenshot 2017-09-16 15 11 27" src="https://user-images.githubusercontent.com/3362943/30509672-5695fe0e-9af1-11e7-83b4-c5d2d88a11f2.png">

<img width="449" alt="screenshot 2017-09-16 15 12 09" src="https://user-images.githubusercontent.com/3362943/30509681-6d817f3a-9af1-11e7-9d25-d9dbebda6ea1.png">